index.js

```diff
@@ -1,81 +1,6 @@
-const { createLogger, format, transports } = require('winston');
-const expressWinston = require('express-winston');
+const { setupNodeLogger } = require('./setup/winston');
+const { setupExpressLogger } = require('./setup/express');
```

Comment:
We need to import the `defaultMatchers` and export it as well `defaultMatchers` so it will be available for the library consumers.
Also, we need to update index.d.ts for the typescript projects. I can help you with that, if you need.

Reply:
I think I got the hang of it. Check now